adapter/redisquota/redisquota.go

```diff
@@ -34,6 +35,9 @@ type redisQuota struct {
 	redisPool *connPool
 
 	redisError error
+
+	// algorithm for rate-limiting, fixed-window or rolling-window
```

Comment:
Maybe add a few lines explaining the difference between the two, pros & cons, etc?